packages/perseus/src/widgets/python-program/python-program.tsx

```diff
@@ -24,12 +24,20 @@ type DefaultProps = {
     height: Props["height"];
 };
 
-/* This renders the program in an iframe. */
-class PythonProgram extends React.Component<Props> {
+/**
+ * This renders the program in an iframe.
+ */
+class PythonProgram extends React.Component<Props> implements Widget {
     static defaultProps: DefaultProps = {
         height: 400,
     };
 
+    iframe = React.createRef<HTMLIFrameElement>();
+
+    getDOMNodeForPath(path: FocusPath) {
```

Comment:
I'm debating if this is a good idea.  In reality, this is not used. But the `Widget` interface has only optional methods _and_ this widget doesn't implement any of them.  As a result, we're triggering Typescript's [weak type detection](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html#weak-type-detection).  Add this function silences the warning without the need for `@ts-expect-error` which I deemed as even worse!